rebasehelper/checkers/abipkgdiff_tool.py

```diff
@@ -89,6 +92,10 @@ def _find_debuginfo(cls, debug, pkg):
     @classmethod
     def run_check(cls, results_dir, **kwargs):
         """Compares old and new RPMs using abipkgdiff"""
+        # Check if ABI changes occured
+        cls.abi_changes = None
+        cls.results_dir = os.path.join(results_dir, 'abipkgdiff')
```

Comment:
Again, you should use `cls.CMD`. Thinking about it, it would probably make more sense to rename it to `cls.NAME`.